doc/admin-guide/files/records.config.en.rst

```diff
@@ -2050,6 +2050,27 @@ Cache Control
 
    The maximum age allowed for a stale response before it cannot be cached.
 
+.. ts:cv:: CONFIG proxy.config.http.cache.guaranteed_min_lifetime INT 0
+   :reloadable:
+   :overridable:
+
+   Establishes a guaranteed minimum lifetime boundary for object freshness.
+   When heuristics are used, and the :ts:cv:`proxy.config.http.cache.heuristic_lm_factor`
+   aging factor is applied, the final minimum age calculated will never be
+   lower than the value in this variable. If heuristics are not used,
+   objects younger than this value are considered stale.
```

Comment:
So, is there actually any difference with heuristics enabled or not? If not, why even bring up the two as separate, rather just say that these impose bounds?

Also, I think for both of these it'd be good to point out that setting these to 0 turns off the feature(s).

Reply:
Good point, PR updated.